config/settings.yml

```diff
@@ -484,6 +484,8 @@ va_mobile:
 
 lighthouse:
   api_key: fake_key
+  facilities:
```

Comment:
Shouldn't the facilities endpoint use its own api-key instead of sharing the one that other endpoints are using?

Reply:
The original idea was to expose the api key when initializing the client so that multiple teams (with separate keys in the config) could use the facilities endpoint. That feels like overkill at this point, so I am going change this to just use a single api key for the client.